modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -8749,6 +8839,76 @@ public void testPostSiteScopedCustomObjectEntryInUserGroup()
 			_OBJECT_FIELD_VALUE_1, jsonObject.getInt(_OBJECT_FIELD_NAME_1));
 	}
 
+	@FeatureFlags("LPD-37056")
+	@Test
+	public void testPostValidate() throws Exception {
+		ObjectField textObjectField = _objectFieldLocalService.getObjectField(
+			_objectDefinition5.getObjectDefinitionId(),
+			_OBJECT_FIELD_NAME_TEXT + "b");
+
+		String error = RandomTestUtil.randomString();
+
+		ObjectValidationRule objectValidationRule =
```

Comment:
Can you improve coverage by implementing a test scenario where the object definition has more than one ObjectValidationRule?